src/components/Facepile/Facepile.Props.ts

```diff
@@ -16,6 +16,9 @@ export interface IFacepileProps extends React.Props<Facepile> {
   /** Show add person button */
   showAddButton?: boolean;
 
+  /** Show persona details when only one face is present in the Facepile */
+  personaDetailsShown?: boolean;
```

Comment:
That makes sense.  I only added it in because I was reading ideas for what the persona could do.  Didn't think about just using the IPersonaProps to do it.  I like that way better.